src/vdaf/prio3.rs

```diff
@@ -1700,11 +1723,14 @@ mod tests {
 
     #[test]
     fn test_prio3_sum() {
-        let prio3 = Prio3::new_sum(3, 16).unwrap();
+        let bits = 16;
+        let max_measurement = 35_891;
+
+        let prio3 = Prio3::new_sum(3, bits, max_measurement).unwrap();
 
         assert_eq!(
-            run_vdaf(CTX_STR, &prio3, &(), [0, (1 << 16) - 1, 0, 1, 1]).unwrap(),
-            (1 << 16) + 1
+            run_vdaf(CTX_STR, &prio3, &(), [0, max_measurement - 2, 0, 1, 1]).unwrap(),
+            max_measurement
```

Comment:
This would test the edge case of the largest valid measurement.
```suggestion
            run_vdaf(CTX_STR, &prio3, &(), [0, max_measurement, 0, 1, 1]).unwrap(),
            max_measurement + 2
```